addons/pos_self_order/controllers/self_entry.py

```diff
@@ -7,7 +7,7 @@
 
 class PosSelfKiosk(http.Controller):
     @http.route(["/pos-self/<config_id>", "/pos-self/<config_id>/<path:subpath>"], auth="public", website=True, sitemap=True)
-    def start_self_ordering(self, config_id=None, access_token=None, table_identifier=None):
+    def start_self_ordering(self, config_id=None, access_token=None, table_identifier=None, subpath=None):
```

Comment:
why do you need this diff?

Reply:
Hello @vlst-odoo 
This diff is meant to prevent this warning: `pos_self_order.controllers.self_entry.start_self_ordering> called ignoring args {'subpath'}`

Reply:
Hello @vlst-odoo 
Reminder!